ouroboros-network-framework/io-tests/Test/Ouroboros/Network/Subscription.hs

```diff
@@ -536,104 +536,109 @@ prop_send_recv
     -> Property
 prop_send_recv f xs _first = ioProperty $ withIOManager $ \iocp -> do
 
-    let first = Socket.AF_INET6
-    let lr = LookupResultIO (Right [0]) (Right [0]) first 1
-        serverPortMap = M.fromList [((Socket.AF_INET, 0), 6062), ((Socket.AF_INET6, 0), 6062)]
+    let lr = LookupResultIO (Right [0]) (Right [0]) Socket.AF_INET6 1
+
```

Comment:
```suggestion
+        hints = Just $ Socket.defaultHints {Socket.addrSocketType = Socket.Stream}
 
-    responderAddress0:_ <- Socket.getAddrInfo Nothing (Just "127.0.0.1") Nothing
+    responderAddress0:_ <- Socket.getAddrInfo hints (Just "127.0.0.1") Nothing
 
-    initiatorAddr4:_ <- Socket.getAddrInfo Nothing (Just "127.0.0.1") (Just "0")
-    initiatorAddr6:_ <- Socket.getAddrInfo Nothing (Just "::1") (Just "0")
+    initiatorAddr4:_ <- Socket.getAddrInfo hints (Just "127.0.0.1") (Just "0")
+    initiatorAddr6:_ <- Socket.getAddrInfo hints (Just "::1") (Just "0")

```

On OSX the default protocol is UDP which won't work.

Reply:
Thanks @karknu.